notes/models.py

```diff
@@ -32,7 +32,7 @@ class Note(TimeStampedModel):
     A simple model to handle adding arbitrary numbers of notes to an animal profile.
     """
     topic=models.ForeignKey(Topic)
-    date=models.DateField(_('Date'), default=datetime.now())
+    date=models.DateField(_('Date'), default=date.today)
```

Comment:
Why change the default from a `datetime` to a `date`?

Reply:
Because it's a `DateField` not a `DateTimeField`.

Reply:
Fair enough.